client-react/src/pages/app/app-settings/GeneralSettings/LinuxStacks/LinuxStacks.tsx

```diff
@@ -127,6 +128,23 @@ const LinuxStacks: React.FC<PropsType> = props => {
     }
   };
 
+  const setInitialData = () => {
+    if (siteStateContext.isWordPressApp) {
+      const initialLinuxFxVersion = initialValues?.config?.properties?.linuxFxVersion;
+      if (initialLinuxFxVersion) {
+        const updatedLinuxFxVersion = CommonConstants.WordPressLinuxFxVersionsMapping[initialLinuxFxVersion.toLocaleLowerCase()];
+        if (updatedLinuxFxVersion) {
+          setFieldValue('config.properties.linuxFxVersion', updatedLinuxFxVersion);
+        }
+      }
+    }
+  };
+
+  useEffect(() => {
+    setInitialData();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [initialValues.config.properties.linuxFxVersion]);
+
```

Comment:
Consider inlining `setInitialData` since it is not used elsewhere.

```suggestion
  useEffect(() => {
    if (siteStateContext.isWordPressApp) {
      const initialLinuxFxVersion = initialValues?.config?.properties?.linuxFxVersion;
      if (initialLinuxFxVersion) {
        const updatedLinuxFxVersion = CommonConstants.WordPressLinuxFxVersionsMapping[initialLinuxFxVersion.toLocaleLowerCase()];
        if (updatedLinuxFxVersion) {
          setFieldValue('config.properties.linuxFxVersion', updatedLinuxFxVersion);
        }
      }
    }
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [initialValues?.config?.properties?.linuxFxVersion]);
```